src/webob/acceptparse.py

```diff
@@ -89,6 +90,17 @@ def _list_1_or_more__compiled_re(element_re):
     )
 
 
+class HeaderState(enum.Enum):
+    """
+    The state of an accept-style header to assist in identifying scenarios
+    an application may want to know about during accept-negotation.
```

Comment:
Typo in "accept-negotation"?

The docstring seems a bit unclear - it may be helpful to explain that an Accept header may have a valid value, an invalid value, or may be missing from the request; and that this enum represents which of the three states the header is in?